DQM/L1TMonitor/src/L1TRPCTF.cc

```diff
@@ -32,15 +32,6 @@ L1TRPCTF::L1TRPCTF(const ParameterSet& ps)
 
   if(verbose_) cout << "L1TRPCTF: constructor...." << endl;
 
-  outputFile_ = ps.getUntrackedParameter<string>("outputFile", "");
-  if ( !outputFile_.empty() ) {
-    cout << "L1T Monitoring histograms will be saved to " << outputFile_.c_str() << endl;
-  }
-
-  bool disable = ps.getUntrackedParameter<bool>("disableROOToutput", false);
```

Comment:
If this parameter is not used anymore it should also be removed from the config files.